### PR TITLE
feat(team): add exec for 2019/2020

### PR DESCRIPTION
**Pull request by EthanWaite**
* add exec for 2019/2020
* move waterstons event to the past

Also might be cool to add email addresses to this page (like secretary[@]compsoc.tech). I'll add those later.

### CODE REVIEW

File: views/team.pug [hunk 1]
```diff
@@ -21,113 +21,126 @@ block content
...
+                          .col-sm-3.no-padding
+                            img(src="./img/team/henry.jpg").exec-photo
                 .row
-                  .col-md-6
+                  .col-sm-6
                     .panel.panel-info
                       .panel-heading
                         h3.position Treasurer
-                        h3.exec-name Jessica Ng
+                        h3.exec-name Merry
```
Comment (robzwolf) [inline views/team.pug]: Does Merry have a surname? 😋

File: public/data/events.json [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
-            "text": [
-              "We are seeing AI everywhere in technology, from taking better pictures on your phone, to self-driving cars. Waterstons present an introduction to AI, through everyone’s favourite game: Rock, Paper, Scissors, Waterbomb, Dynamite.",
-              "Compete in teams to build the ultimate game playing AI, and then challenge the other teams’ AIs to see who has mastered the game.",
-              "Waterstons developers will be on hand to help you learn how to code the AI, setup, and deploy. Any programming language will be suitable."
-            ]
-          }
-        ],
+        "upcoming": [],
         "past": [
+            {
```
Comment (robzwolf) [inline public/data/events.json:5]: Good shout on moving this to `events.past` – except the homepage doesn't use `events.json`, so the Waterstons event is still showing on the homepage. Maybe just remove it from the homepage for now?

**Review by robzwolf — CHANGES_REQUESTED** [review]
Great stuff – have just left a couple of small comments. 😃

File: views/team.pug (hunk 1 above)
Comment (EthanWaite) [reply views/team.pug]: Derp, let's give her a surname :)

File: public/data/events.json (hunk 2 above)
Comment (EthanWaite) [reply public/data/events.json:5]: kk, removed it for now since the next event hasn't been announced yet

**Review by robzwolf — APPROVED** [review]
Approved 😁

**Pull request merged**